### PR TITLE
Fixed various grammar mistakes

### PR DESCRIPTION
**Pull request by Bitwise101**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,7 +128,7 @@ Some additional documentation about raylib design can be found in raylib GitHub
 contact and networks
 ---------------------
 
-raylib is present in several networks and raylib community is growing everyday. If you are using raylib and enjoying it, feel free to join us in any of these networks. The most active network is our [Discord server](https://discord.gg/raylib)! :)
+raylib is present in several networks and the raylib community is growing everyday. If you are using raylib and enjoying it, feel free to join us in any of these networks. The most active network is our [Discord server](https://discord.gg/raylib)! :)
```
Comment (raysan5) [inline README.md:131]: Not required. Please remove `the`.

**Comment by Bitwise101** [comment]
@raysan5 I have updated it to follow your wishes.

File: README.md [hunk 2]
```diff
@@ -116,7 +116,7 @@ learning and docs
 
 raylib is designed to be learned using [the examples](https://github.com/raysan5/raylib/tree/master/examples) as the main reference. There is no standard API documentation but there is a [**cheatsheet**](https://www.raylib.com/cheatsheet/cheatsheet.html) containing all the functions available on the library a short description of each one of them, input parameters and result value names should be intuitive enough to understand how each function works. 
 
-Some additional documentation about raylib design can be found in the raylib GitHub Wiki. Here are the relevant links:
+Some additional documentation about raylib design can be found in   raylib GitHub Wiki. Here are the relevant links:
```
Comment (raysan5) [inline README.md:119]: It seems there are some extra spaces in the sentence. Please, could you review it.

**Comment by raysan5** [comment]
@Bitwise101 thanks for the review!

**Pull request merged**